docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -379,4 +379,4 @@ ANALYZE TABLE [catalog_name.][db_name.]table_name PARTITION(partcol1[=val1] [, p
 
 *NOTE:* For the fix length types (like `BOOLEAN`, `INTEGER`, `DOUBLE` etc.), we need not collect the `avgLen` and `maxLen` from the original records.
 
-{{< top >}}
\ No newline at end of file
+{{< top >}}
```

Comment:
What is the purpose of this change?

Reply:
This line of code should not be changed, sorry for my carelessness. I will undo this line.